tyk-docs/assets/scss/home.scss

```
@@ -198,8 +199,8 @@ ol, ul {
 
 @media (min-width: 992px) {
   .section-padding {
-    padding-left: 100px;
-    padding-right: 90px;
+    padding-left: 1%;
+    padding-right: 1%;
```

here you can also use `padding-inline`
```suggestion
    padding-inline: 1%;
```